media-gfx/photivo/patches/photivo-0~pre20240904.patchset

```diff
@@ -42,39 +42,7 @@ index 8f4557b..f395ae2 100644
 2.51.0
 
 
-From 07e0f2b8d8e482db9c2b8a00a31cf6776519f121 Mon Sep 17 00:00:00 2001
-From: Gerasim Troeglazov <3dEyes@gmail.com>
-Date: Sat, 21 Nov 2020 23:15:28 +1000
-Subject: Set username
-
-
-diff --git a/Sources/ptMain.cpp b/Sources/ptMain.cpp
-index 2be0eb7..e85e2cb 100644
---- a/Sources/ptMain.cpp
-+++ b/Sources/ptMain.cpp
-@@ -526,6 +526,9 @@ int photivoMain(int Argc, char *Argv[]) {
-   InitializeMagick(*Argv);
- 
-   //QApplication TheApplication(Argc,Argv);
-+#ifdef Q_OS_HAIKU
-+  Qstring    user = "Photivo_user";
-+#else
-   QStringList environment = QProcess::systemEnvironment();
-   QString user = environment.filter(
-       QRegularExpression("^USERNAME=|^USER=", QRegularExpression::CaseInsensitiveOption)).first();
-@@ -534,6 +537,7 @@ int photivoMain(int Argc, char *Argv[]) {
-       user="_"+user.right(user.length()-l-1);
-   }
-   user="photivo"+user;
-+#endif
- 
- #ifdef Q_OS_MAC
-   TheApplication = new MyQApplication(user, Argc,Argv);
--- 
-2.51.0
```

Comment:
This failed on previous run at the buildmaster, should probably get an update for it.
For now the update should fix building with new exiv2.

Reply:
https://build.haiku-os.org/buildmaster/master/x86_64/logviewer.html?buildruns/13570/builds/86511.log